src/main/java/net/sf/mzmine/parameters/parametertypes/PasswordParameter.java

```diff
@@ -83,7 +87,7 @@ public void setValue(String newValue) {
   public boolean checkValue(Collection<String> errorMessages) {
     if (!valueRequired)
       return true;
-    if ((value == null) || (value.trim().length() == 0)) {
+    if ((value == null) || (value.isEmpty())) { //white spaces can be valid in pw. So no trim
```

Comment:
It's true that pw can contain white space, but don't you think a password that consists only of whitespace should be considered invalid? :)

Reply:
I thought that it is a bit strange to check the password requirements of third party applications.
However, all whitespace passwords are a strange edge case. I reverted the change.